src/hooks/useDisableInputUntilReply.ts

```diff
@@ -8,6 +8,7 @@ interface UseDisableInputUntilReplyProps {
   lastMessage?: SendableMessage;
   botUser?: Member;
   currentUserId: string | null;
+  blockInputWhileBotResponding?: boolean | number;
```

Comment:
Q. This prop doesn't sound it can have a number value. Which case can it be a number?

Reply:
its used as timeout value to force unblock blocked message input in case reply is never received
https://sendbird.slack.com/archives/C0585965FFA/p1716798555294749?thread_ts=1716795864.100449&cid=C0585965FFA

Reply:
currentUserId 랑 blockInputWhileBotResponding 같은 경우 파라메터로 받지 않고
훅 안에서 context 접근해서 바로 쓸 수 있을것 같습니다.